src/tests/SearchForm.test.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import SearchForm from "../components/SearchForm";
+
+describe("SearchForm", () => {
+  const validProps = {
+    searchText: "Test Search Text",
+    setSearchText: jest.fn(),
+    onSubmit: jest.fn(),
+  };
+
+  test("renders", () => {
+    const { asFragment } = render(
+      <SearchForm
+        searchText={validProps.searchText}
+        setSearchText={validProps.setSearchText}
+        onSubmit={validProps.onSubmit}
+      />
+    );
+
+    expect(asFragment()).toMatchSnapshot();
+  });
```

Comment:
Another test could be added here, how about checking if the function passed into the component is being called when the button is clicked? 
```
  test("clicking button causes submit function to be called", () => {
    render(
      <SearchForm
        searchText={validProps.searchText}
        setSearchText={validProps.setSearchText}
        onSubmit={validProps.onSubmit}
      />
    );
    const button = screen.getByText("Search");
    fireEvent.click(button);

    expect(validProps.onSubmit).toHaveBeenCalledTimes(1);
  });
```